tools/explorer/lib/tests/ontology_wrapper_test.py

```diff
@@ -0,0 +1,106 @@
+"""Testing module for ontology.py"""
+from absl.testing import absltest
+
+from yamlformat.validator.external_file_lib import RecursiveDirWalk
+from yamlformat.validator import presubmit_validate_types_lib
+from yamlformat.validator import namespace_validator as nv
+
+from lib.ontology_wrapper import OntologyWrapper
+from lib.model import EntityTypeField
+from lib.tests import test_constants
+
+class OntologyTest(absltest.TestCase):
+
+  def setUp(self):
+    super(OntologyTest, self).setUp()
+    self.yaml = RecursiveDirWalk(test_constants.ONTOLOGY_ROOT)
+    self.config = presubmit_validate_types_lib.SeparateConfigFiles(self.yaml)
+    self.universe = presubmit_validate_types_lib.BuildUniverse(self.config)
+    nv.NamespaceValidator(self.universe.GetEntityTypeNamespaces())
+    self.ontology = OntologyWrapper(self.universe)
+
+  def testGetAllFieldsForTypeName(self):
+    expected_output = [
+        EntityTypeField('', 'zone_use_label', True),
```

Comment:
add a comment here what is it, 
for example:
`EntityTypeField('', 'zone_use_label`, /*foo=*/True),`